features/step_definitions/et1_atos_export_steps.rb

```diff
@@ -14,30 +21,40 @@
 
 Then(/^my entered data is formatted into an XML format file and is available for ATOS to download$/) do
   within_admin_window do
+    api = EtFullSystem::Test::AdminApi.new
+    api.login
+    expect { api.get_claimants }.to eventually include a_hash_including(first_name: user.dig(:personal, :first_name))
     admin_pages.jobs_page.run_export_claims_cron_job
   end
   expect { atos_interface }.to eventually have_zip_file_containing(:et1_claim_xml_for, user: user), timeout: 30, sleep: 2
 end
 
-
 Then(/^my uploaded ET1 CSV data is formatted into a TXT format file and is available for ATOS to download$/) do
   within_admin_window do
+    api = EtFullSystem::Test::AdminApi.new
```

Comment:
One for the future - maybe we could refactor this to just read 

```
expect { admin_api.get_claimants }.to eventually include a_hash_includin
```

without the other 2 lines ?  admin_api could be a global method the same as 'autos_interface' - and it could be a singleton that tracks its own login state and logs in if needed before doing the action - would be really efficient too